index.html

```diff
@@ -46,5 +42,189 @@ <h3 class="logo">John Doe </h3>
     <div class="bckimg">
         <img class="sideimg"src="images/illustration-header-mobile.png" alt="illustration">
     </div>
+    
+     <!--Second Branch: Recent Work and About me Page-->
+<section class="workSection">
+    <div>
+    <p class="workSecPara">My Recent Works</p>
+    
+    <img class="workSectionRec" src="images/rec2.png" alt="Rec2">
+</div>
+
+<div>
```

Comment:
- I think it is important to properly indent your divs as they are all inside the `<section class="workSection">` section of your portfolio. I'd like you to apply this to all divs in your work section 👍🏽 

- I think it will be better for your `My Recent Works` to be in an `hx` tag (h2 - h5) because this is a heading and not a paragraph. It can be something like this 👉🏽 `<h2 class="workSecPara">My Recent Works</h2>` 🌸